src/Microsoft.AspNet.SignalR.Client/Transports/ServerSentEventsTransport.cs

```diff
@@ -107,7 +109,9 @@ internal void OpenConnection(IConnection connection, string data, CancellationTo
                 _stop = true;
 
                 // This will no-op if the request is already finished.
-                ((IRequest)state).Abort();
+                var request = state as IRequest;
+                if (request != null)
```

Comment:
I don't know when it would be null. I was attempting to fix this issue: https://github.com/SignalR/SignalR/issues/3376 . I was also trying to make the file consistent; I believe that there are null checks for it elsewhere in the file.